test/zdtm/static/vdso01.checkskip

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+grep "\[vdso\]" /proc/self/maps -q
```

Comment:
nit: grep -q "\[vdso\]" /proc/self/maps 
￼

Reply:
err, I think backslashes there are important :)

Reply:
You are right and I suggest moving -q before non-option arguments. You have not seen backslashes in my comment because I didn't take into account that this is markdown.

Reply:
I moved the `-q` and force pushed.